multihost_test/bz_automation/test_automation.py

```diff
@@ -127,3 +138,31 @@ def test_2091062(self, multihost, create_localusers):
         client.logout()
         assert "pam_motd: error scanning directory" not in \
                execute_cmd(multihost, "cat /var/log/secure").stdout_text
+
+    def test_2228934(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: Using "pam_access", ssh login fails with this entry in
+            /etc/security/access.conf "+:username:localhost server1.example.com"
+        :id: 5b669434-35b9-11ee-b8b3-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2228934
+        :steps:
+            1. Enable "with-pamaccess" feature using authselect
+            2. Configure  /etc/security/access.conf
+            3. Try to log in with the user
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+        """
+        client = multihost.client[0]
+        file_location = "/multihost_test/bz_automation/script/bz824858.sh"
```

Comment:
What's the purpose of this script? The name isn't meaningful and the test case description doesn't explain what it does

Reply:
Now that I've read the complete test case I understand what it does. IMO, it would be better to try to login explicitly in the test case instead of relying in an external script.

Reply:
Can't you use `client.run_command("ssh user@localhost")` ?

Reply:
it will not work, terminal will stuck there . As now the new terminal will act as user not as root from where the test is running

Reply:
ok